streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -146,7 +166,7 @@
      * @return a {@link KStream} for the specified topics
      */
     public <K, V> KStream<K, V> stream(final Serde<K> keySerde, final Serde<V> valSerde, final String... topics) {
-        return stream(null, keySerde, valSerde, topics);
+        return stream((AutoOffsetReset) null, keySerde, valSerde, topics);
```

Comment:
`return stream(null, null, keySerde, valSerde, topics);`

Do the call directly instead of the cast.